src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -387,15 +389,33 @@ export default class PlotsTab extends React.Component<IPlotsTabProps,{}> {
             return PotentialViewType.MutationTypeAndCopyNumber;
         }
 
+        // one axis is clinical attributes and the other is treatment
+        if (this.showClinicalAttrsWithTreatments) {
+            //  establish whether data may contain limit values
+            // (for now only supported for treatment data)
+            if (this.limitValuesCanBeShown) {
+                return PotentialViewType.LimitValMutationTypeAndCopyNumber;
+            }
+            return PotentialViewType.MutationTypeAndCopyNumber;
+        }
+        
         //  establish whether data may contain limit values
         // (for now only supported for treatment data)
         if (this.limitValuesCanBeShown) {
             return PotentialViewType.LimitVal;
         }
+        
 
         // neither axis gene or treatment
         return PotentialViewType.None;
     }
+    
+    @computed get showClinicalAttrsWithTreatments() {
```

Comment:
I dont love the name. I think since its a computed and not an action, the verb should be the 'ing' tense, rather than imperative. Also neither clinical attr nor treatment should be plural.

overall I would suggest the name `isClinicalAttrVsTreatment`